.github/workflows/core.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         cd mobile && ./bazelw test \
+            --build_tests_only \
```

Comment:
just curious - was this needed or just an improvement?

Reply:
it avoids trying to build mobile/test/common/jni/BUILD on the build we rm -rf the listener file :-P